### ISSUE TITLE
Typo at comment in values.yaml in autoInstrumentation section

### ISSUE THREAD

**Issue by timdeluxe** [grafana/k8s-monitoring-helm#997]
Here is a typo: https://github.com/grafana/k8s-monitoring-helm/blob/v1.6.13/charts/k8s-monitoring/values.yaml#L141 
It says "Enable gathering Kubernetes Pod logs." but it looks like that is just copied from https://github.com/grafana/k8s-monitoring-helm/blob/v1.6.13/charts/k8s-monitoring/values.yaml#L122

Should be fixed to say something about enabling auto instrumentation.

**Comment by petewall** [comment]
Ahh yes! Thanks for catching that.

**Comment by Abhishek022001** [comment]
can i work on this issue?

**Comment by petewall** [comment]
Absolutely! Update the values.yaml file and run make build to update Readme.md

**Comment by petewall** [comment]
Fixed in fff0bf0f
